migration_toolkit_for_containers/about-mtc.adoc

```diff
@@ -1,5 +1,5 @@
 [id="about-mtc"]
-= About the {mtc-full}
+= About the Migration Toolkit for Containers
```

Comment:
Would you mind adding a period at the end of line 8, please?

Reply:
Good catch